### DISCUSSION TITLE
Global css doesn't apply on home page

### DISCUSSION THREAD

**Discussion by nour-alsatari** [vercel/next.js#23248] [Help]
I'm new to Next.js and as im going through this great example and applying everything step by step, i noticed global styles don't apply on index.js any reason why? could it be because index.js has it's own styling using <style jsx> which overrides the global? 
https://nextjs.org/learn/basics/assets-metadata-css/global-styles

**Comment by leerob** [comment]
Hey! Global styles do apply to `index.js`, but you are correct that global styles can be overridden if styled-jsx is used on the page-level 👍
